Tony/box_office_challenge.py

```diff
@@ -0,0 +1,70 @@
+# this variable is assigned a list of strings
+top_50_list = ['Star Wars: Episode I - The Phantom Menace', 'The Sixth Sense', 'Austin Powers: The Spy Who Shagged Me', 'Toy Story 2', 'The Matrix', 'Tarzan', 'Big Daddy', 'The Mummy', 'Runaway Bride', 'The Blair Witch Project', 'Notting Hill', 'The World Is Not Enough', 'Double Jeopardy', 'Wild Wild West', 'Analyze This', "The General's Daughter", 'American Pie', 'Inspector Gadget', 'Shakespeare in Love', 'Sleepy Hollow', 'The Haunting', 'Patch Adams', 'Entrapment', 'Pokémon: The First Movie - Mewtwo Strikes Back', 'Payback', 'Deep Blue Sea', 'American Beauty', 'The Thomas Crown Affair', 'Stuart Little', 'Blue Streak', 'The Green Mile', 'Bowfinger', 'Life', 'The Bone Collector', "She's All That", 'End of Days', 'Three Kings', 'A Civil Action', 'Stepmom', 'Eyes Wide Shut', 'Never Been Kissed', 'Forces of Nature', 'Varsity Blues', 'Message in a Bottle', "You've Got Mail", 'South Park: Bigger, Longer & Uncut', 'Stigmata', 'Life Is Beautiful', 'The Prince of Egypt',
+               'Deuce Bigalow: Male Gigolo']
+
+print(top_50_list)
+
+
+# TODO: Take the string "movies_str" and convert it into a list and assign it to a variable called "remaining_50_list"
+# Note: Print your new variable to see the list of movies
+remaining_50_list = movies_str.split(',')
+print(sorted(remaining_50_list))
+print()
+
+# TODO: Hmm. Looks like "remaining_50_list" contains the movie "The Matrix" which is also in the "top_50_list". Let's remove this duplicate title from "remaining_50_list".
+# Note: Make sure to double check the list you are removing from!
+
+setTop = set(top_50_list)
+setRemaining = set(remaining_50_list)
+
+setAll = setTop.union(setRemaining)
+setClean = setAll - setTop
+
+remaining_50_list = list(setClean)
+
+print(sorted(remaining_50_list))
+
+print()
+
+# TODO: We're down to 49 movies in "remaining_50_list". Let's add a movie to the list called "Anywhere But Here"
+# Note: Print the variable "remaining_50_list" and see if the movie has been added!
+
+remaining_50_list.append("Anywhere But Here")
+print(sorted(remaining_50_list))
+
+print()
+
+# TODO: Let's make sure "remaining_50_list" has exactly 50 movie titles
+print(len(remaining_50_list))
+
+print()
+
+# TODO: Great! Let's join the two lists into one list called "top_100".
+# Note: Print your new variable to see the combined list of movies
+print()
+
+top100 = top_50_list + remaining_50_list
+
+# TODO: Let's print the first movie in our "top_100" list. Let's use the index lookup method to do this.
+
+print(top100[0])
+
+print()
+
+# TODO: Let's print the top 10 movies in "top_100" as a list. Let's use the slice syntax to accomplish this!
+
+print(top100[0:10])
+
+print()
+
+# TODO: Let's convert the list "top_100" back into a string. Assign it to a variable called "top_100_str".
+# Note: Print your new variable to see the string version of the movies
+
+top_100_str = str(top100).strip('[]')
```

Comment:
This works, but to convert the list back to a string without the quotes you can do this: `top_100_str = ','.join(top100)`